htlcswitch/link.go

```diff
@@ -2012,7 +1996,7 @@ func (l *channelLink) processRemoteAdds(fwdPkg *channeldb.FwdPkg,
 			continue
 		}
 
-		heightNow := l.bestHeight
+		heightNow := atomic.LoadUint32(&l.cfg.Switch.bestHeight)
```

Comment:
Rather than accessing a _private_ member, the links should access a public method of the switch. This way, the details (the atomic load), don't need to be leaked to the lower layers.

Reply:
Maybe make retrieve it using a new method `BestHeight()` set in the channel link config.

Reply:
Fixed to use the a `BestHeight` method, although I'm not sure we need to set it in the config since we already have a reference to the switch itself in there.